composer.json

```diff
@@ -15,17 +15,16 @@
         }
     ],
     "require": {
-        "php": "^5.3.9|^7.0",
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/finder": "~2.3|~3.0",
+        "php": "^5.3.9|^7.0|^7.1",
```

Comment:
this change should not be necessary afaik

Reply:
you're right, I removed `|^7.1`